website/cue/reference/components/sinks/elasticsearch.cue

```diff
@@ -117,7 +117,7 @@ components: sinks: elasticsearch: {
 				options: {
 					region: {
 						common:      true
-						description: "The [AWS region](\(urls.aws_regions)) of the target service. This defaults to the region named in the endpoint parameter, or the value of the `$AWS_REGION` or `$AWS_DEFAULT_REGION` environment variables if that cannot be determined, or \"us-east-1\"."
+						description: "The [AWS region](\(urls.aws_regions)) of the target service. This defaults to the region named in the endpoints parameter, or the value of the `$AWS_REGION` or `$AWS_DEFAULT_REGION` environment variables if that cannot be determined, or \"us-east-1\"."
```

Comment:
How does this defaulting work if there are multiple endpoints, possibly in different AWS regions?

Reply:
This option is only used for AWS authentication when it's selected as a strategy, and it's required for that. From what I was able to follow in the code, vector and dependencies, we are never extracting region from uri of an endpoint, so I'm guessing that if an endpoint with a different region in uri than the one specified here is used it will be rejected from AWS as not being signed well since the request will be for different region then the one specified.

It should be possible to parse out region from an endpoint during config building and default to that, else use this field.

Reply:
Ah I see. Should we remove the statement that it will extract the region from the URI then?

Reply:
Yes, but also the rest of the statement, since none of that is being done. So something like
> The [AWS region](\(urls.aws_regions) of the target service. Used for AWS authentication.

Reply:
It's now:
`The AWS region for AWS authentication.`